From 0e6859c98a90a1b44abc6f29ba64628afc1be7cb Mon Sep 17 00:00:00 2001
From: q66 <daniel@octaforge.org>
Date: Sat, 25 Jul 2020 17:29:23 +0200
Subject: [PATCH 3/4] mkinitrd-dracut.sh: use vmlinux regex for ppc*, vmlinuz
 for i686

Previously this would not catch ppc64le, now it does; same with
i686.
---
 mkinitrd-dracut.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git mkinitrd-dracut.sh mkinitrd-dracut.sh
index 82d90684..1ea3e514 100755
--- mkinitrd-dracut.sh
+++ mkinitrd-dracut.sh
@@ -57,10 +57,10 @@ default_kernel_images() {
         s390|s390x)
             regex='image'
             ;;
-        ppc|ppc64)
+        ppc*)
             regex='vmlinux'
             ;;
-        i386|x86_64)
+        i*86|x86_64)
             regex='vmlinuz'
             ;;
         arm*)
-- 
2.27.0

